### ISSUE TITLE
Upgrade addons from 14.x 

### ISSUE THREAD

**Issue by rasmuswoelk** [facebook/react#6548]
After upgrading to react `15.0.1` I get the following error when running `npm install`

    npm WARN react-addons-css-transition-group@0.14.8 requires a peer of react@^0.14.8 but none was installed.
    npm WARN react-addons-update@0.14.8 requires a peer of react@^0.14.8 but none was installed.

I think it is due to these two add-ons I include:

    "react-addons-css-transition-group": "^0.14.7",
    "react-addons-update": "^0.14.7",

I would intuitively upgrade them to

    "react-addons-css-transition-group": "^0.15.1",
    "react-addons-update": "^0.15.1",

But since it is a "breaking change" in version, I'd like to check up the changelog with each addon to ensure that I do not have any code that will break. However, I cannot find any changelog or gitrepo belonging to theese add-ons. What is the right approach here? Should we simply follow the react version number every time?

Thanks in advance; a bit lost here.

**Comment by ev1stensberg** [comment]
[Changelog master](https://github.com/facebook/react/commits/master ) No real change to css or addons as I see. Just small patches on the interal module styling and the docs along with some performance cleanup regarding conditionals etc. Here's the closest thing I found [PR](https://github.com/facebook/react/commit/531205065f52cf17c6465e262a1d090d33a8a0ff)
[PR2](https://github.com/facebook/react/commit/a7af9c6ce453229504fa00b2360b5af9f896eb46)
[PR3](https://github.com/facebook/react/commit/30dc842e346cad7879b7b2ac1cf6f0c1c9234357)
[PR4](https://github.com/facebook/react/commit/a5c164dacad2597a1685a3ab401b1a3b7ff1f3eb)
[PR5](https://github.com/facebook/react/commit/fb13cf55feb3391af30fb8c9e45b6c2aec40c6ae)
[PR6](https://github.com/facebook/react/pull/6195)
[PR7](https://github.com/facebook/react/commit/0f1240e8772be73342797172d29734994adf0874)

PR merges are from 30 back in time, where they released v15. I don't think they break things without documenting "breaking" changes, so yeah, just watch the blog posts, and they usually inform you! 

Even

**Comment by ev1stensberg** [comment]
FWIW you could just update it and revert the npm via github or do it manually to see if something breaks.

**Comment by gaearon** [comment]
>I would intuitively upgrade them to
>"react-addons-css-transition-group": "^0.15.1",

This would be 15.0.1, as we jumped to a major version now.

>However, I cannot find any changelog or gitrepo belonging to theese add-ons

[React 15 blog post](http://facebook.github.io/react/blog/2016/04/07/react-v15.html) lists the complete changelog, including the addons. We don’t actively work on addons so you shouldn’t expect breaking changes there. I looked through the blog post, and it seems that we made no changes to either of those addons.
